docs/src/apps/rent.md

```diff
@@ -1 +1,46 @@
-# Accounts and Rent
+# Storage Rent for Accounts
+
+Keeping accounts alive incurs storage cost called _rent_ because the cluster must actively maintain the data to process any future transactions on it. This is not like Bitcoin or Ethereum, where account storages don't incur any costs.
+
+The rent is debited from an account's balance by the runtime upon the first access in the current epoch by transactions or once per an epoch if there are no transactions. The rent is paid upfront for the next epoch. If an account can't pay rent for the next epoch, it's purged immediately at the start of a new epoch. The fee is currently a fixed rate, measured in bytes-times-epochs. The fee may change in the future.
+
+Accounts can be exempt from paying rent by maintaining a minimum balance, this rent-exemption is described below.
+
+## Calculation of rent
+
+Note: The rent rate can change in the future. And this applies to the testnet and mainnet-beta.
+
+As of writing, the fixed rent fee is 19.055441478439427 lamports per byte-epoch. And an epoch is roughly 2 days.
```

Comment:
```suggestion
As of writing, the fixed rent fee is 19.055441478439427 lamports per byte-epoch on the testnet and mainnet-beta clusters. An [epoch](../terminology.md#epoch) is roughly 2 days.
```

Reply:
Users may wonder about this strange number (especially as lamports can't be split). How is this calculated?

Reply:
Ok, I'll document this as well. :)

Reply:
done!